ops/gulp/docs-js.js

```diff
@@ -0,0 +1,27 @@
+const gulp = require('gulp');
+const gp_concat = require('gulp-concat');
+const gp_rename = require('gulp-rename');
+const gp_uglify = require('gulp-uglify');
+const gulpif = require('gulp-if');
+const gulp_jslint = require('gulp-jslint');
+
+let environment = require('../lib/environment');
+
+const paths = {
+    src: './src/scripts',
+    dest: './www/js'
+}
+
+let techneDestJS = environment.production ? 'techne-all.min.js' : 'techne-all.js';
+
+const task = (cb) => {
+    return gulp.src([`${paths.src}/*`])
+        .pipe(gulp_jslint())
+        .pipe(gp_concat('concat.js'))
```

Comment:
I'm not sure concatenation will work here. This will take all the files in the directory and concat them in alpha order, right? This may not be a problem depending on how we organize the files but we may need to control the order of the files, for example, if we are including jQuery here it would have to be included first.